bundles/catalogue/metadatacatalogue/publisher/MetadataSearchTool.js

```diff
@@ -50,13 +59,11 @@ Oskari.clazz.define('Oskari.mapframework.publisher.tool.MetadataSearchTool',
         * @returns {Object} tool value object
         */
         getValues: function () {
-            const me = this;
-
-            if (me.state.enabled) {
-                var json = {
+            if (this.handler?.getState()?.allowMetadata) {
+                let json = {
```

Comment:
`const` should work here too since the object is modified without reassigning the variable itself.